### PR TITLE
Updated tests for resampling Generic Map and some documentation in the Sunpy/data/test.

### PR DESCRIPTION
**Pull request by prateekiiest**
@Cadair 
I have updated the keys for generic map (included naxis1 and naxis2).
This PR is done to clear out the error causing in PR #2010

### CODE REVIEW

**Comment by prateekiiest** [comment]
@Cadair I added some documentation  related to sunpy.data.test along with issue #732 . 
Pls tell me if something extra needs to be documented here, then I will send further PRs for other Sunpy.databse.test

File: sunpy/net/vso/vso.py [hunk 1]
```diff
@@ -705,13 +705,17 @@ def download(self, method, url, dw, callback, errback, *args):
 
     @staticmethod
     def by_provider(response):
+        """Returns a dictionary of provider corresponding
```
Comment (Cadair) [inline sunpy/net/vso/vso.py]: This should not be formatted like this, it's a docstring not code.

Something like this would be correct. In general the triple quote marks `"""` should be on their own lines.
```python
def by_provider(response):
    """
    Returns a dictionary of provider corresponding
    to records in the response. 
    """

**Comment by prateekiiest** [comment]
OK :)

File: sunpy/map/tests/test_mapbase.py [hunk 2]
```diff
@@ -365,7 +365,7 @@ def test_resample_metadata(generic_map, sample_method, new_dimensions):
     assert resampled_map.meta['crval2'] == generic_map.center.y.value
     for key in generic_map.meta:
         if key not in ('cdelt1', 'cdelt2', 'crpix1', 'crpix2',
-                       'crval1', 'crval2'):
+                       'crval1', 'crval2', 'naxis1', 'naxis2'):
```
Comment (Cadair) [inline sunpy/map/tests/test_mapbase.py]: Why is this change here? As I commented in #2010 it should not be needed.

Comment (prateekiiest) [inline sunpy/map/tests/test_mapbase.py]: Actually, I had changed these things in my forked repo and I didn't delete the previous changes. As a result, when I am opened a PR for this issue, all the previous changes are being shown with it.

File: _config.yml [hunk 3]
```diff
@@ -0,0 +1 @@
+theme: jekyll-theme-merlot
```
Comment (Cadair) [inline _config.yml]: errrr, why is this here?!

Comment (prateekiiest) [inline _config.yml]: @Cadair ,Some patch problems might have occurred.

File: _config.yml [hunk 4]
```diff
@@ -0,0 +1 @@
+
```
Comment (Cadair) [inline _config.yml]: This file should not be here at all.

File: sunpy/net/vso/vso.py [hunk 5]
```diff
@@ -705,13 +705,21 @@ def download(self, method, url, dw, callback, errback, *args):
 
     @staticmethod
     def by_provider(response):
+        """
```
Comment (Cadair) [inline sunpy/net/vso/vso.py]: You just opened another PR with this change in, it should only be in one place.

File: _config.yml (hunk 4 above)
Comment (prateekiiest) [inline _config.yml]: yes, its a file in the sunpy.github.io in the website making repository. 
You can close this PR.
2 or 3 PRs has been collectively addressed here 
I will send separate PRs for the other related issues :)

File: sunpy/net/vso/vso.py (hunk 5 above)
Comment (prateekiiest) [inline sunpy/net/vso/vso.py]: yes, I have sent a seperate PR for this. This PR is all jumbled up with other PRs, due to some patch mismatch. You can close this PR.
